pkg/client/typed/dynamic/client.go

```diff
@@ -177,6 +204,29 @@ func (rc *ResourceClient) Watch(opts v1.ListOptions) (watch.Interface, error) {
 		Watch()
 }
 
+// Get gets the resource with the specified name.
+func (src *SubresourceClient) Get(name string) (*runtime.Unstructured, error) {
+	result := new(runtime.Unstructured)
+	err := src.get(name).
+		SubResource(src.subresource).
+		Do().
+		Into(result)
+	return result, err
+}
+
+// Update updates the provided subresource.
+func (src *SubresourceClient) Update(obj *runtime.Unstructured) (*runtime.Unstructured, error) {
+	result := new(runtime.Unstructured)
+	req, err := src.update(obj)
+	if err != nil {
+		return result, err
+	}
+	err = req.SubResource(src.subresource).
+		Do().
+		Into(result)
+	return result, err
+}
```

Comment:
Scale doesn't require post., so I'd prefer to defer that since we're already post-branch.